openapi.yml

```diff
@@ -46,6 +46,44 @@ components:
       name: api_key
       in: query
   schemas:
+    Holding:
+      description: location holdings information
+      type: object
+      properties:
+        location:
+          type: string
+        call_number:
+          type: string
+        status:
+          type: string
+    Identifiers:
+      description: standardized identifiers
+      type: object
+      properties:
+        isbns:
+          type: array
+          items:
+            type: string
+        issns:
+          type: array
+          items:
+            type: string
+        doi:
+          type: string
+    Link:
```

Comment:
I think we should consider what we want out of links. Are we just looking for a bag o' undifferentiated links? Are we looking for machine actionable links? There may be existing classifications of links we could use like the rel types for microformats or something. What we want to do with them should probably drive how we model them.

Reply:
Yes, exactly the discussion I am looking forward to having. I'm not sure we'll have a decision in the short term so maybe we can open a ticket to revisit the Link model and schedule a time to discuss it.

Reply:
Yeah, I think it's fine to just merge as is and we can adjust later.

Reply:
The ontology includes defined terms for some links (such as [http://id.loc.gov/ontologies/bibframe.html#p_electronicLocator](http://id.loc.gov/ontologies/bibframe.html#p_electronicLocator) for a link to the item itself), but other links will be harder to classify and may just require a local controlled vocabulary

Reply:
I'm adding that to the ticket I opened to discuss this further:
https://mitlibraries.atlassian.net/browse/DIP-119